### DISCUSSION TITLE
Becoming a Drep. how do I "Register as a DRep with an on-chain registration certificate"?

### DISCUSSION THREAD

**Discussion by CharityDA0** [IntersectMBO/cardano-node#5529] [General]
I was checking http://sancho.network/roles/drep/ and was wondering if there were instructions to complete step 4.

**Comment by ntfreem** [comment]
Hello

Thank you for your inquiry, the instructions you are looking for can be found here: https://sancho.network/tutorials/drep-registration

If you need support its possible to open a request https://iohk.zendesk.com/hc/en-us/requests/new

Thank you
